### PR TITLE
Assignment 8 files

### PR DESCRIPTION
**Pull request by sdlipp**
Circle class and tests

### CODE REVIEW

File: students/ScottL/session08/circle_class.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        return cls(diameter / 2)
+
+    def __str__(self):
+        return f"Circle with a radius of: {self.radius}"
+
+    def __repr__(self):
+        return f"Circle({self.radius})"
+
+    def __add__(self, other):
+        return f"Circle({self.radius + other.radius})"
```
Comment (mdgregor) [inline students/ScottL/session08/circle_class.py]: This should result in a `Circle()` object, not a string value

File: students/ScottL/session08/circle_class.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    def __mul__(self, other):
+        return f"Circle({self.radius * other})"
```
Comment (mdgregor) [inline students/ScottL/session08/circle_class.py]: same here.

File: students/ScottL/session08/circle_class.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        return f"Sphere with a radius of: {self.radius}"
+
+    def __repr__(self):
+        return f"Sphere({self.radius})"
+
+    def __add__(self, other):
+        return f"Sphere({self.radius + other.radius})"
+
+    def __mul__(self, other):
+        return f"Sphere({self.radius * other})"
```
Comment (mdgregor) [inline students/ScottL/session08/circle_class.py]: these two as well

File: students/ScottL/session08/circle_class.py (hunk 1 above)
Comment (sdlipp) [reply students/ScottL/session08/circle_class.py]: ahh yeah that makes more sense, i'll get that adjusted

**Pull request merged**